cocos/ui/UIButton.cpp

```diff
@@ -788,12 +789,17 @@ void Button::setTitleFontSize(float size)
     {
         _titleRenderer->setSystemFontSize(size);
     }
-    else
+    else if (_type == FontType::TTF)
     {
         TTFConfig config = _titleRenderer->getTTFConfig();
         config.fontSize = size;
         _titleRenderer->setTTFConfig(config);
     }
+    else
+    {
+        // BMFont can't set its font size
+        size = _fontSize;
```

Comment:
@andyque so what your suggestion this function should do when user invoke the setTitleFontSize and use BMFont? Should do nothing at all? or we can use assert there?

Thanks,